test/createTest.m

```diff
@@ -34,40 +34,51 @@ function createTest()
     fprintf(FID, start);
     
     %Make a list of all json names stored locally.
-    files=dir([fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'jsons','*.json')]);
+    files=dir([fullfile(fileparts(mfilename('fullpath')), '..', 'test', 'data', 'jsons','*.json')]);
     jsonFileList={files.name};
     jsonFileNameList = strrep(jsonFileList, '.json', '');
 
     %For each json, create a test.
     
     for i = 1:length(jsonFileList)      
         %Find the json download location.
-        json = spm_jsonread(fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'jsons', jsonFileList{i}));
-        [designMatrix, dmLocation] = searchforType('nidm_DesignMatrix', json.x_graph);
+        json = spm_jsonread(fullfile(fileparts(mfilename('fullpath')), '..', 'test', 'data', 'jsons', jsonFileList{i}));
+        
+        % Deal with sub-graphs (bundle)
```

Comment:
The bundle is part of the NIDM-Results specification: http://nidm.nidash.org/specs/nidm-results_130.html#dfn-nidm_0000027.